flagevalmm/evaluator/clip_sim_evaluator.py

```diff
@@ -0,0 +1,73 @@
+import torch
```

Comment:
remove the file, you can use or modify CLIPScoreEvaluator `flagevalmm/evaluator/clip_score_evaluator.py`